packages/video_player/video_player/example/lib/main.dart

```diff
@@ -262,6 +289,16 @@ class _BumbleBeeRemoteVideoState extends State<_BumbleBeeRemoteVideo> {
       ),
     );
   }
+
+  Future<void> setupSubtitle() async {
+    if (widget.useAssetCC) {
```

Comment:
what's the difference between the `if` and `else` here? Is subtitle the same as CC?